docs/source/data/file-uploads.mdx

```diff
@@ -5,29 +5,35 @@ description: Enabling file uploads in Apollo Server
 
 There are a few ways to accomplish file uploads in Apollo Server. Our [blog post](https://www.apollographql.com/blog/backend/file-uploads/file-upload-best-practices/) on file uploads goes in depth on the approaches we've taken, outlining the pros and cons of each.
```

Comment:
Honestly I would just remove this file. I think that graphql-upload is essentially broken by design; people who want to use it are welcome to use graphql-upload's own docs.  Maybe we can add a small section in the CSRF prevention saying "by the way if you decide to use graphql-upload, you will find that it gets blocked by the CSRF Prevention feature; instead of disabling CSRF prevention (which is really helping you here!) pass the `{'Apollo-Require-Preflight': 'true'}` header to `createUploadLink`".

But otherwise I feel like one major version after we drop the direct integration is long enough to keep around the documentation of a bad feature that has mostly been responsible for security issues and where we've rewritten our own best practices to say not to use it at all.

Reply:
Makes sense to me! I'll delete this file and make sure we add a note to the CORS article.